Fix dfb-example

Order of inclusion gives a strange effect on the definition of
gst_debug_log. Swapping the include statements solve it.

Not upstreamed since there will be no more updates to this version
of gst-plugins-bad.

Signed-off-by: Arnout Vandecappelle (Essensium/Mind) <arnout@mind.be>
---
diff -rup gst-plugins-bad-0.10.23/ext/directfb/dfb-example.c gst-plugins-bad-0.10.23.fixed/ext/directfb/dfb-example.c
--- gst-plugins-bad-0.10.23/ext/directfb/dfb-example.c	2011-12-11 19:49:11.000000000 +0100
+++ gst-plugins-bad-0.10.23.fixed/ext/directfb/dfb-example.c	2013-05-13 00:32:00.507469138 +0200
@@ -1,7 +1,7 @@
 
-#include <directfb.h>
 #include <stdio.h>
 #include <gst/gst.h>
+#include <directfb.h>
 
 static IDirectFB *dfb = NULL;
 static IDirectFBSurface *primary = NULL;
